doc/old/image_data_handlers.rst

```diff
@@ -89,8 +89,9 @@ handled by the given packages:
 | :sup:`1` *only with JpegImagePlugin*
 | :sup:`2` *only with Jpeg2KImagePlugin*
 | :sup:`3` *only if (0028,0100) Bits Allocated = 8*
-| :sup:`4` *with the pylibjpeg-libjpeg plugin*
-| :sup:`5` *with the pylibjpeg-openjpeg plugin*
+| :sup:`4` *with the pylibjpeg-rle plugin and using the* :meth:`~pydicom.dataset.Dataset.decompress` *method*
```

Comment:
One suggestion here - since there are multiple options for RLE, adding something about the speed improvement here would be helpful.